pub/errors/processor.php

```diff
@@ -480,7 +480,7 @@ protected function _setReportData($reportData)
      */
     public function saveReport($reportData)
     {
-        $this->reportData = $reportData;
+        $this->reportData = $reportData . PHP_EOL;
```

Comment:
Since `$this->reportData` gets serialized, this fails to add a newline to the end of the file. It needs to be moved a few lines down, to the `file_put_contents` call.

Reply:
Oh interesting. Ok, I will move it and redo the PR.